### PR TITLE
[ASIA-3582] add locale and show beta logo for id

### PR DESCRIPTION
**Pull request by deepakwiser**
## Context

We are launching a new route for source currency IDR. we will be beta testing for some time and hence we need to show the beta logo

### Changes

image

## Checklist
- [x] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: items/navigation.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "items": [
+    {
+      "isBeta": true,
```
Comment (deepakwiser) [inline items/navigation.json:4]: is there an existing way to display this text in a box, and move it left 

requirement
image

Comment (danlaush) [reply items/navigation.json:4]: I don't believe there is. Until now the `items` array is for linking to features, not just displaying information—does that make sense? Did this go through design? The screenshot makes it feel a bit like TransferWise as a company is in beta, not just the route.

Comment (twrdjpalmer) [reply items/navigation.json:4]: @deepakwiser could you pick this up with @LozWilliams please, as we're in the process redesigning the nav

**Comment by karanjthakkar** [comment]
Closing since a lot has changed for v8. Please open a new PR if you need to do this change.

**Pull request closed without merging**